source/tags/Button/README.md

```diff
@@ -1,7 +1,16 @@
 # Button
 
-## API
+Buttons are used to denote user interaction. They can be links, or static elements.
```

Comment:
The comma is unnecessary. And it's' unclear to me what's meant by `static elements`.